xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -88,10 +88,6 @@
   margin-bottom: 0;
 }
 
-textarea#content {
-  width: 100%;
```

Comment:
How is this change related to this PR?

Reply:
Note that even before this change, this was unnecessary because repetitive with standard form style rules.